tests/integration/test_named_collections/configs/users.d/users_no_default_access2.xml

```diff
@@ -0,0 +1,10 @@
+<clickhouse>
```

Comment:
maybe we can call this file users_with_access_management.xml or users_with_no_default_access_with_access_management.xml , what do you think ?